Podcast/EpisodeSubjectView.swift

```diff
@@ -159,10 +159,10 @@ class EpisodeSubjectView: UIView {
         episodeUtilityButtonBarView.playButton.addTarget(self, action: #selector(didPressPlayButton), for: .touchUpInside)
     }
     
-    convenience init(episode: Episode) {
-        self.init()
-        setup(with: episode)
-    }
+//    convenience init(episode: Episode) {
```

Comment:
why is this commented out?

Reply:
I'm deleting it that's why. No one used it and we don't want it.